CMakeLists.txt

```diff
@@ -43,6 +43,10 @@ endif()
 
 add_compile_options(-Wall -Wextra)
 
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-fpermissive>)
```

Comment:
IIRC fixing this would require a lot of changes to irods's API that I'm not sure we actually want to make, so probably more permanent, but I'm not sure.

Reply:
Okay, that sounds right. Resolving